frameworks/react/__tests__/AdvancedPicture.test.tsx

```diff
@@ -54,11 +57,12 @@ describe('AdvancedPicture', () => {
       expect(component.html()).toContain(
         '<picture>' +
         '<img src="https://res.cloudinary.com/demo/image/upload/sample">' +
-        '<source srcset="https://res.cloudinary.com/demo/image/upload/c_crop,w_500/dog" sizes="80vw"' +
-        ' media="(min-width: 500px) and (max-width: 700px)">' +
-        '<source srcset="https://res.cloudinary.com/demo/image/upload/c_crop,w_1000/woman" sizes="80vw"' +
-        ' media="(min-width: 1000px) and (max-width: 2000px)">' +
-        '</picture>');
+        '<source media="(min-width: 500px) and (max-width: 700px)" sizes="80vw" ' +
```

Comment:
note that `sizes` doesn't do anything if the resource itself does not contain a size descriptor, like this:

`src='https://res.cloudinary.com/demo/image/upload/e_sepia/c_scale,w_1366/sample 1366w'`

no harm, but also no effect.

Reply:
This is for the case where autoOptimalBreakpoints is off. Do we want to remove sizes or add a size descriptor @const-cloudinary?

Reply:
@patrick-tolosa `sizes` was explicitly provided in the input.

Reply:
We can ignore this, but it does nothing. 
As mentioned, no harm here.